### PR TITLE
Implement polymorphic serialization for includes

### PR DESCRIPTION
**Pull request by jshow**
Implement polymorphic serialization for includes

### CODE REVIEW

File: spec/lib/object_serializer_spec.rb [hunk 1]
```diff
@@ -251,12 +251,6 @@ def advertising_campaign
         expect(movies_serialized).to include(movie.id)
       end
     end
-
-    it 'polymorphic throws an error that polymorphic is not supported' do
```
Comment (lassiter) [inline spec/lib/object_serializer_spec.rb]: Are you planning on replacing with tests for polymorphic relationships?

**Comment by shishirmk** [comment]
@jshow could you add some tests please. I can merge it in once tests are added

**Comment by jshow** [comment]
will do @shishirmk @lassiter - was looking for feedback on impl first. cheers

**Comment by jshow** [comment]
Just getting into writing tests, but appears that polymorphic doesn't have any present tests ? 

Doesn't seem possible, but scanning the code it appears to be so

Gonna take me a quite a bit longer to reconstruct the polymorphic spec - can I have some help beyond the docs ?

**Comment by lassiter** [comment]
If I recall, there is a context called `groupee` but there are no unit tests.

**Comment by shishirmk** [comment]
@jshow we went with different pull request. So closing this. Polymorphic includes are now supported and Released in [Version 1.4](https://github.com/Netflix/fast_jsonapi/releases/tag/1.4)

**Comment by jshow** [comment]
thanks folks, I got buried

**Pull request closed without merging**